setup/management.sh

```diff
@@ -32,6 +32,9 @@ done
 apt_install duplicity python-pip virtualenv certbot
 hide_output pip2 install --upgrade boto
 
+# b2sdk is installed outside the pipenv, so it can be used by duplicity
+hide_output pip3 install b2sdk
```

Comment:
The latest version of duplicity (pulled from their ppa) depends on `python3` istd. of `python2`.

```
vagrant@mailinabox:~$ sudo apt-cache depends duplicity
duplicity
  Depends: python3
  Depends: python3
  Depends: python3-fasteners
  Depends: python3-future
  Depends: <python3:any>
    python3
  Depends: libc6
  Depends: librsync1
  Depends: gnupg
  Depends: lftp
  Depends: python3-pexpect
  Suggests: ncftp
  Suggests: python3-boto
  Suggests: python3-paramiko
  Suggests: python3-kerberos
```

You can delete L33 and install `boto` via pip3 here as well:

```suggestion
hide_output pip3 install --upgrade b2sdk boto
```
The comment in L22 also needs an update now.